test/integration/filters/local_reply_during_decoding_filter.cc

```diff
@@ -20,6 +20,16 @@ class LocalReplyDuringDecode : public Http::PassThroughFilter {
                                        "");
     return Http::FilterHeadersStatus::StopIteration;
   }
+
+  Http::FilterDataStatus decodeData(Buffer::Instance&, bool) override {
+    ASSERT(false);
```

Comment:
nit:add explanatory comment that this method should never be invoked in tests, same for one below